File: Java/Inserindo.java
import java.sql.Connection;
import java.sql.DriverManager;
import java.sql.PreparedStatement;

public class Inserindo {

    public static void main(String[] args) {
        try {
            System.out.println("Conectando com o MySQL ...");
            Class.forName("com.mysql.jdbc.Driver");
            String url = "jdbc:mysql://localhost:3306/aula_bd";

            Connection mydb = DriverManager.getConnection(url, "root", "root");
            
            String sql = "insert into estado (id, nome, sigla) values (?, ?, ?)";
            PreparedStatement ps = mydb.prepareStatement(sql);
            ps.setInt(1, 6);
            ps.setString(2, "Tocantins");
            ps.setString(3, "TO");
            int registros = ps.executeUpdate();
            System.out.println("Registros afetados: " + registros);
        } catch (Exception e) {
            System.out.println("Erro: " + e);
        }
    }
}

File: Python/inserindo.py
import mysql.connector

print('Conectando com o MySQL ...')

try:
    mydb = mysql.connector.connect(
    host="localhost",
    user="root",
    password="root",
    database="aula_bd"
    )
    
    sql = "insert into estado (id, nome, sigla) values (%s, %s, %s)"
    estado = (6, "Tocantins", "TO")

    cursor = mydb.cursor()
    cursor.execute(sql, estado)
    mydb.commit()
    print("Inserido: ", estado)
    print("Registros afetados: ", cursor.rowcount)
except BaseException as e:
    print("Erro: ", e)
else:
    mydb.close()